pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -77,6 +88,14 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
             // PercentileRawTDigest
             String percentileString = remainingFunctionName.substring(10);
             return new PercentileRawTDigestAggregationFunction(firstArgument, parsePercentileToInt(percentileString));
+          } else if (remainingFunctionName.matches("KLL\\d+")) {
+            // PercentileKLL
+            String percentileString = remainingFunctionName.substring(3);
```

Comment:
why 3? can you add an comment?

Reply:
3 is the number of characters already consumed ("KLL"). All the other if branches uses the same logic so I didn't think it requires extra explanation.